### PR TITLE
[MI-3554] Did some minor improvements in the ConvertToMD function and added new unit tests

### PR DESCRIPTION
**Pull request by manojmalik20**

### CODE REVIEW

File: server/markdown/markdown.go [hunk 1]
```diff
@@ -6,13 +6,24 @@ import (
 	"github.com/mattn/godown"
 )
 
+var stringsToCheckHTML = map[string]bool{
+	"<div":  true,
+	"<p ":   true,
```
[the remark this answers is not in the document]
Comment (manojmalik20) [reply server/markdown/markdown.go:11]: Technically, only `<div` and `<p` are enough for covering all the cases. We're using `<p ` and `<p>` to increase the specificity in this case.

**Review by ayusht2810 — APPROVED** [review]

File: server/markdown/markdown.go [hunk 2]
```diff
@@ -6,13 +6,24 @@ import (
 	"github.com/mattn/godown"
 )
 
+var stringsToCheckHTML = []string{
```
Comment (Kshitij-Katiyar) [inline server/markdown/markdown.go:9]: Maybe ?
```suggestion
var stringsToCheckInHTML = []string{
```

File: server/markdown/markdown_test.go [hunk 3]
```diff
@@ -14,14 +14,19 @@ func TestConvertToMD(t *testing.T) {
...
 		},
 		{
 			description:    "Text contains div and paragraph tags",
 			text:           "This is text area with <div> and <p> tags",
 			expectedOutput: "This is text area with \n and \n tags\n\n\n\n\n",
 		},
+		{
+			description:    "Text contains paragraph and image tags",
+			text:           "This is text area with <img src=''> and <p class=''>Paragraph</p> tags",
+			expectedOutput: "This is text area with ![]() and \nParagraph\n\n\n tags\n",
```
Comment (Kshitij-Katiyar) [inline server/markdown/markdown_test.go:28]: Where this tags word came in expectedOutput from the actual text ?

Comment (manojmalik20) [reply server/markdown/markdown_test.go:28]: `tags` is present in the text itself.

**Review by Kshitij-Katiyar — APPROVED** [review]

**Pull request merged**